python/play.py

```diff
@@ -0,0 +1,96 @@
+import pyarrow as pa
```

Comment:
Does this need to be included in this PR?  If so, we should rename it and move it in a benchmark somwhere.  Although I'd rather just keep it out.

Reply:
oh, sorry, will remove it. I keep `git rm` this file, then I get some lint issues then I `git add .` it back haha

Reply:
fixed, thanks.